services/ops/EmailerAgent/README.md

```diff
@@ -0,0 +1,105 @@
+Emailer
+=======
+
+The Emailer agent allows an instance of the VOLTTRON platform to send
+email.  When used in combination with the Alert agent alerts from
+unpublished configured devices will automatically be sent.  In addition,
+agents are able to send emails directly through the pubsub interface.
+
+Agents needing to send an email through the instance can do so by
+sending the following header and message to the "emailer" topic.  The
+Emailer agent monitors the "platform_send_email" topic.  The following 
```

Comment:
It isn't clear to me from this why there is an "emailer' and "platform_send_email" topic....